Core/GDCore/Tools/FileStream.h

```diff
@@ -24,7 +24,7 @@ namespace gd {
 class GD_CORE_API FileStream : public std::iostream {
  public:
 #if defined(WINDOWS) && __GLIBCXX__
-  using InternalBufferType = __gnu_cxx::stdio_filebuf<char>;
+  using InternalBufferType = std::basic_filebuf<char>;
```

Comment:
Not sure about that, but I couldn't compile the tests without that change.

Reply:
Let's assume it's fine, I've not compiled tests on Windows since a while so it might be a necessary update.